packages/zql/src/mutate/mutator-registry.ts

```diff
@@ -55,26 +59,26 @@ import {
  * ```
  */
 export function defineMutators<
-  S extends Schema,
-  C,
   T extends MutatorDefinitions<S, C>,
+  S extends Schema = DefaultSchema,
```

Comment:
I will also address this - `defineMutators` should not use the default type params